tests/test_srclist_by_beam.py

```diff
@@ -19,10 +19,8 @@ def test_srclist_by_beam():
     for obs in obsids:
         for sn in source_numbers:
             for slist in source_lists:
-                print('srclist_by_beam.py', '-m', '{}.metafits'.format(obs), '-n',
-                      sn, '-s', slist)
                 # Run srclist
-                subprocess.call(['srclist_by_beam.py', '-m',
+                subprocess.call(['python3', 'srclist_by_beam.py', '-m',
```

Comment:
As above, but the main script is an executable, so calling python manually should be redundant.

Reply:
Wasn't working without it which was probably due to the space in the shebang!